lib/widget/radial_gauge_chart .dart

```diff
@@ -4,95 +4,95 @@ import 'package:syncfusion_flutter_gauges/gauges.dart';
-                        GaugeRange(
-                            startValue: 8,
-                            endValue: 11,
-                            color: Colors.yellow,
-                            startWidth: 30,
-                            endWidth: 30),
-                        GaugeRange(
-                            startValue: 11,
-                            endValue: 20,
-                            color: Colors.green,
-                            startWidth: 30,
-                            endWidth: 30)
-                      ],
-                      pointers: <GaugePointer>[
-                        NeedlePointer(
-                            value: 12,
-                            lengthUnit: GaugeSizeUnit.logicalPixel,
-                            needleLength: 70)
-                      ],
-                      annotations: <GaugeAnnotation>[
-                        GaugeAnnotation(
-                            widget: Container(
-                                child: Text(
-                              'ผลคะแนน\n12 คะแนน',
-                            )),
-                            angle: 90,
-                            positionFactor: 0.5)
-                      ])
-                ]),
-            Text('การแปลผล',
-                style: Theme.of(context).textTheme.bodyText1,
-                textAlign: TextAlign.left),
-            Text('12 คะแนนขึ้นไป = mild independence',
-                style: Theme.of(context).textTheme.bodyText1,
-                textAlign: TextAlign.center),
-            Text('9-11 คะแนน= moderately independence',
-                style: Theme.of(context).textTheme.bodyText1,
-                textAlign: TextAlign.center),
-            Text('5-8 คะแนน = severe independence',
-                style: Theme.of(context).textTheme.bodyText1,
-                textAlign: TextAlign.center),
-            Text('0-4 คะแนน = total independence',
-                style: Theme.of(context).textTheme.bodyText1,
-                textAlign: TextAlign.center),
-          ],
-        ),
+  Widget build(BuildContext context) => Column(
+        crossAxisAlignment: CrossAxisAlignment.stretch,
+        children: [
+          SfRadialGauge(
+            enableLoadingAnimation: true,
+            animationDuration: 4000,
+            axes: <RadialAxis>[
+              RadialAxis(
+                  axisLineStyle: AxisLineStyle(
+                    thickness: 50,
+                    /*For gradient color*/
+                    // thicknessUnit: GaugeSizeUnit.factor,
+                    // gradient: SweepGradient(colors: <Color>[
+                    //   Colors.red,
+                    //   Colors.orange,
+                    //   Colors.yellow,
+                    //   Colors.green
+                    // ], stops: <double>[
+                    //   0.10,
+                    //   0.25,
+                    //   0.50,
+                    //   1
+                    // ]),
```

Comment:
If this is not needed anymore. remove it. else use //TODO to remind things

Reply:
I wait for a consult today and change to the final dashboard pattern